alpheios_nemo_ui/data/templates/main/container.html

```diff
@@ -56,46 +57,69 @@
     {% include "main::footer.html" %}
   {% endblock %}
   <script type="text/javascript">
-    document.addEventListener("DOMContentLoaded", function(event) {
-      import ('/assets/nemo.secondary/js/alpheios-embedded.min.js').then(embedLib => {
-        console.info(`Embedded library has been imported successfully`)
+    window.onerror = function(message) {
+      var parser = new UAParser();
+      var browserData = parser.getBrowser();
+      var allowedBrowsers = ['Chrome', 'Firefox', 'Mobile Safari', 'Safari'];
 
-        window.AlpheiosEmbed.importDependencies(
-          { mode: 'custom',
-            libs: { components: '/assets/nemo.secondary/js/alpheios-components.min.js' } }).then(Embedded => {
-          const embedded = new Embedded(
-            {
-               clientId:'alpheios-reader',
-               authEnv: serverEnv,
-               enabledSelector:'.entry-content',
-               popupInitialPos: {
-                 left: '50px',
-                 top: '230px'
-               },
-              toolbarInitialPos: {
-                top: '220px',
-                right: '15px'
-              },
-              layoutType: 'readingTools',
-              {% if collections['current'] and collections['current']['id'] and 'greek' in collections['current']['id'] %}
-              textLangCode: 'grc'
-              {% else %}
-              textLangCode: 'lat'
-              {% endif %}
-            })
-          embedded.activate().then(function () {
-            import ('/assets/nemo.secondary/js/alpheios-embed-support.js')
-              .then(m => m.embedPostActivation(embedded))
-              .catch(e => console.error(`There is an error importing alpheios-embed-support.js:`, e))
+      if (allowedBrowsers.indexOf(browserData.name) === -1) {
+        var messageNew = 'Alpheios may not work with your browser. Please use Chrome, Firefox or Safari to access this site.';
```

Comment:
I would add the words "latest version" before a browser list (so that it'll be "Please use the latest version of  Chrome, Firefox or Safari ...") to indicate that the newer version of the browser will be the better our code will (supposedly) work. We're using all the latest technologies so this can be important for us. What do you think?

Reply:
@balmas ,  should I add "latest version" - I agree with Kirill, but it is your text :)

Reply:
yes, I think it's a good idea.